plugin/src/main/java/com/exadel/aem/toolkit/plugin/metadata/scripting/Annotated.java

```diff
@@ -20,33 +20,53 @@
 import java.util.stream.Collectors;
 
 import org.apache.commons.lang3.StringUtils;
+import org.mozilla.javascript.Callable;
 
 import com.exadel.aem.toolkit.core.CoreConstants;
 
+/**
+ * Represents a mix-in object used with {@link AbstractAdapter}-based objects to expose Java objects leveraging
+ * reflective operations to
```

Comment:
I tried to understand this sentence 5 times but failed to do so) 
perhaps it can be rewritten to be more human-readable?

Reply:
Thanks) It was a glitch. a broken phrase